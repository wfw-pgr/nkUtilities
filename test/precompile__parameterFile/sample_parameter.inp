
$ This is a parameter file for some calculation

$ <define>  @var1 = 0.1
$ <include> filepath = "include.inp"
$$$ <include> filepath = "test/precompile__parameterFile/include.inp"

BEGIN{

  TITLE    = @title

  x,y,z    = ( @x, @y, @z )
  particle = @particleType
  iterMax  = @iterMax

}END