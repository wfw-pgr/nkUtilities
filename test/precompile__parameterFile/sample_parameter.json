{

    x: 0.0,
    y: 1.0,
    z: 2.0,
    iterMax:10,
    "particle.type":"electron", 

}
