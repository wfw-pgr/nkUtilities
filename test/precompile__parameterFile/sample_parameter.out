
$ This is a parameter file for some calculation

$ <define>  @var1 = 0.1
$$$ <include> filepath = "include.inp"
$ <include> filepath = "test/precompile__parameterFile/include.inp"

$ <loadJSON> filepath = "test/precompile__parameterFile/sample_parameter.json"
$ <define> @var2 = 0.2

$$$ <loadJSON> filepath = "sample_parameter.json"
$$$ <define> @var2 = 0.2

BEGIN{

  TITLE    = TEST01

  x,y,z    = ( 0.0, 1.0, 2.0 )
  particle = electron
  iterMax  = 10

}END